mevislab.github.io/content/tutorials/dataobjects/surfaces/surfaceexample1.md

```diff
@@ -13,6 +13,9 @@ menu:
     parent: "surfaces"
 ---
 # Surface Example 1: Create Winged Edge Mesh out of voxel images and CSOs
+
+{{< youtube "-KnZ5a27T0c">}}
+
 ## Introduction
 In this example you will learn how to create a Winged Edge Mesh (WEM). There are several ways how to create WEMs, a few of them are shown in this example. Instead of creating WEMs, it is also possible to load existing WEMs. This was already shown in [Surface Objects (WEM)](tutorials/dataobjects/surfaceobjects) chapter.
```

Comment:
There are several approaches on creating...
There are several ways of creating...
There are several ways to create...

Instead of creating them, you could also load existing ones, as shown in chapter....
Instead of creating WEMs, they can also be imported, see chapter...